examples_tests/49.Radix_Sort/Radix_Sort.cpp

```diff
@@ -159,18 +158,18 @@ void Radix_Sort::Radix_Sort::Init()
 		auto* data = reinterpret_cast<buffer_type*>(downloadStagingArea->getBufferPointer()) + address;
 		auto cpubufferalias = nbl::core::make_smart_refctd_ptr<nbl::asset::CCustomAllocatorCPUBuffer<nbl::core::null_allocator<uint8_t> > >(Buffer_Sizeof, data, nbl::core::adopt_memory);
 
-		auto buffer_after_compute_shader = static_cast<buffer_type*>(cpubufferalias->getPointer());
+		auto buffer_after_compute_shader = reinterpret_cast<buffer_type*>(cpubufferalias->getPointer());
```

Comment:
@devshgraphicsprogramming I have an issue, Allocating for example 10 elements with ascending values from 1 to 9, changing them in comp shader code and after copying memory from GPU to CPU, I am getting array of 0,0,0,0...

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/47/files/d051ae7044998edd1660f9327dbd0da3444a40be..a912f03a9c222ba9d9046c10d70f27aaf330e6b3#r566702972